lib/manageiq/api/engine.rb

```diff
@@ -7,6 +7,21 @@ class Engine < ::Rails::Engine
       config.autoload_paths << root.join('lib', 'services').expand_path
       config.autoload_paths << root.join('lib').expand_path
 
+      config.after_initialize do
+        $api_log.info("Initializing Environment for #{::Api::ApiConfig.base[:name]}")
+        $api_log.info("")
+        $api_log.info("Static Configuration")
+        ::Api::ApiConfig.base.each { |key, val| log_kv(key, val) }
+
+        $api_log.info("")
+        $api_log.info("Dynamic Configuration")
+        ::Api::Environment.user_token_service.api_config.each { |key, val| log_kv(key, val) }
+      end
+
+      def self.log_kv(key, val)
+        $api_log.info("  #{key.to_s.ljust([24, key.to_s.length].max, ' ')}: #{val}")
```

Comment:
We have a thing in VMDBLogger that logs hashes, which might just simplify this...

```
$api_log.log_hashes(::Api::ApiConfig.base)
```